src/MyEventEmitter.js

```diff
@@ -1,12 +1,122 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  validateEventName(eventName) {
+    if (typeof eventName !== 'string') {
+      throw new Error('The event name supposed to be a string');
+    }
+  }
+
+  validateListener(listener) {
+    if (typeof listener !== 'function') {
+      throw new Error('The listener supposed to be a fanction');
```

Comment:
There is a typo in the error message 'The listener supposed to be a fanction', it should be 'function'.